spec/label_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'spec_helper'
+
+RSpec.describe Label do
+  describe '#initialize' do
+    it 'creates a new label with title and color' do
+      label = Label.new('Title', 'Color')
+      expect(label.title).to eq('Title')
+      expect(label.color).to eq('Color')
+      expect(label.items).to be_an(Array)
+      expect(label.items).to be_empty
+    end
+  end
+
+  describe '#add_label' do
+    it 'adds a book to the label items' do
+      label = Label.new('Title', 'Color')
+      book = double('Book')
+      label.add_label(book)
+      expect(label.items).to include(book)
+    end
+  end
```

Comment:
- [ ] As required for the project, all public methods in all classes to have unit tests, kindly ensure that you also add tests to the `add_item()` method in the `Label` class after it has been implemented